docs/development/portable_demo.rst

```diff
@@ -0,0 +1,20 @@
+Portable SecureDrop Demo
+========================
+
+When at a conference or travelling, it is possible to prepare a
+SecureDrop demo using portable hardware and adapted usage scenarios.
+
+Hardware
+--------
+
+* A laptop running the :doc:`staging virtual environment <virtual_environments>`
+* A Tails compatible laptop with a physical radio kill switch
+* Four USB keys prepared for the staging environment running on the laptop
+  * Transfert
+  * Journalist
+  * SVS
+  * Admin
```

Comment:
These nested bullets aren't rendering well:

![screen shot 2018-02-12 at 11 12 22 am](https://user-images.githubusercontent.com/7832803/36114719-aa636918-0fe5-11e8-93d6-612c33c081d8.png)

Adding a line before `*Transfer` should resolve

Reply:
done